packages/cactus-common/src/main/typescript/signer-key-pairs.ts

```diff
@@ -0,0 +1,26 @@
+import crypto from "crypto";
+import secp256k1 from "secp256k1";
+
+export interface ISignerKeyPairs {
+  privateKey: any;
+  publicKey: any;
+}
+
+export class Secp256k1Keys {
```

Comment:
Will it be only option for signing algorithm ?
I hope to change API to allow using another algorithm as well.